cmd/hermes/cmd_eth.go

```diff
@@ -17,33 +17,35 @@ import (
 )
 
 var ethConfig = &struct {
-	PrivateKeyStr   string
-	Chain           string
-	Attnets         string
-	Devp2pHost      string
-	Devp2pPort      int
-	Libp2pHost      string
-	Libp2pPort      int
-	PrysmHost       string
-	PrysmPortHTTP   int
-	PrysmPortGRPC   int
-	DialConcurrency int
-	DialTimeout     time.Duration
-	MaxPeers        int
+	PrivateKeyStr               string
+	Chain                       string
+	Attnets                     string
+	Devp2pHost                  string
+	Devp2pPort                  int
+	Libp2pHost                  string
+	Libp2pPort                  int
+	Libp2pPeerscoreSnapshopFreq int // seconds
```

Comment:
Hehe snapshop :)

Suggestion:

```suggestion
	PeerscoreSnapshots time.Duration
```

Reply:
shame... hahahaha 
right, let me put a `time.duration` directly (saves also de extra parsing of the int)